syntax = "proto3";

service ProphetForecast {
  rpc Forecast(ForecastRequest) returns (ForecastResponse) {}
}

message ForecastRequest {
  bytes csv_data = 1;
  int32 periods = 2;
}

message ForecastResponse {
  oneof result {
    bytes forecast_csv = 1;
    string error = 2;
  }
}
